docs/concepts-replication.md

```diff
@@ -39,24 +40,26 @@ Synchronous geo-replication provides the highest availability and also guarantee
 
 ## Replication patterns
 
-Pulsar provides a great degree of flexibility for customizing your replication strategy. You can set up different replication patterns to serve your replication strategy for an application between multiple data centers.
+Pulsar provides a great degree of flexibility for customizing your replication strategy. You can set up different replication patterns to serve your replication strategy for an application between multiple data centers. 
+
+Pulsar supports the following three replication patterns:
 
 ### Full-mesh replication
 
 Using full-mesh replication and applying the [selective message replication](administration-geo.md#selective-replication), you can customize your replication strategies and topologies between any number of data centers.
 
-![An example of a full-mesh replication pattern](/assets/full-mesh-replication.svg)
+![Example of full-mesh replication pattern in Pulsar](/assets/full-mesh-replication.svg)
 
 ### Active-active replication
 
 Active-active replication is a variation of full-mesh replication, with only two data centers. Producers can run at any data center to produce messages, and consumers can consume all messages from all data centers.
 
-![An example of an active-active replication pattern](/assets/active-active-replication.svg)
+![Example of active-active replication pattern in Pulsar](/assets/active-active-replication.svg)
 
 For how to use active-active replication to migrate data between clusters, refer to [here](administration-geo.md#migrate-data-between-clusters-using-geo-replication).
 
 ### Aggregation replication
 
-The aggregation replication pattern is typically used when replicating messages from the edge to the cloud. For example, assume you have 3 clusters in 3 fronting datacenters and one aggregated cluster in a central data center, and you want to replicate messages from multiple fronting datacenters to the central data center for aggregation purposes. You can then create an individual namespace for the topics used by each fronting data center and assign the aggregated data center to those namespaces.
+The aggregation replication pattern is typically used when replicating messages from the edge to the cloud. For example, assume you have 3 clusters in 3 fronting data centers and one aggregated cluster in a central data center, and you want to replicate messages from multiple fronting data centers to the central data center for aggregation purposes. You can then create an individual namespace for the topics used by each fronting data center and assign the aggregated data center to those namespaces.
```

Comment:
FYI: data center is also correct https://www.techtarget.com/searchdatacenter/definition/data-center#:~:text=A%20data%20center%20%2D%2D%20also,disseminate%20large%20amounts%20of%20data.